python/lsst/ctrl/mpexec/cli/script/build.py

```diff
@@ -0,0 +1,113 @@
+# This file is part of ctrl_mpexec.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from click import ClickException
+
+from ... import CmdLineFwk
+from ...cmdLineParser import (_ACTION_ADD_TASK,
+                              _ACTION_DELETE_TASK,
+                              _ACTION_CONFIG,
+                              _ACTION_CONFIG_FILE,
+                              _ACTION_ADD_INSTRUMENT)
+
+
+def build(config_file=[], config=[], delete=[], instrument=[], order_pipeline=None, pipeline_dot=None,
```

Comment:
Python advises against using mutables as defaults because those mutables can be changed by this function and that would change the defaults to everyone. Since you are only using these for iteration you can use a tuple as a default value.